prediction_market_agent/agents/autogen_general_agent/deploy.py

```diff
@@ -41,9 +41,13 @@ def run(self, market_type: MarketType) -> None:
 
     def get_bets(self, market_type: MarketType) -> list[Bet]:
         one_day_ago = utcnow() - timedelta(days=1)
-        return market_type.market_class.get_bets_made_since(
+        bets = market_type.market_class.get_bets_made_since(
             better_address=APIKeys().bet_from_address, start_time=one_day_ago
         )
+        # filter bets with unique title, i.e. get 1 bet per market
+        seen_titles = {bet.market_question: bet for bet in bets}
```

Comment:
The function is called `get_bets` but it will return only market-unique bets and for the last 1 day.